### DISCUSSION TITLE
Squarespace support for wildcard certificates

### DISCUSSION THREAD

**Discussion by IoSonoPiero** [NginxProxyManager/nginx-proxy-manager#3823] [Q&A]
Hello people,
I'm a newbie, and I discovered Nginx Proxy Manager a few days ago.
I want to add a wildcard SSL certificate, so enabling the toggle for "Use DNS Challenge" in the list of DNS, but I've seen that Squarespace is missing from the list of DNS providers.
My domains migrated from Google to Squarespace, so I need this option.
Maybe I'm missing something, but I repeat myself: I'm a newbie.
I tried to look for this specific item on the web and here, but I need help finding it.
Can someone help me, please?

Thanks,
Piero

**Comment by jre08** [comment]
I am also experiencing a problem with the transfer of domains between google and squarespace.  I received a certificate while my domain was using Google DNS, and now (June 23, 2024) my certificate has expired and during the mean time my domain was transferred to squarespace.  Unfortunately now I can't renew my certificate due to Let's Encrypt not working with squarespace authenticator.
Can someone please point us in the right direction.

**Reply by teotimepacreau** [reply]
same problem here. Can't do a DNS challenge with Squarespace on Nginx Proxy Manager. I'm moving to other provider

**Comment by mrkito** [comment]
i dont understand how Squarespace domain add to nginx)

**Reply by teotimepacreau** [reply]
Can't work, you must setup Cloudflare Domain Name Server that proxy your Squarespace domain in order to work

**Comment by ceddy30** [comment]
Use Cloudflare as your public DNS for your domain. 

Same situation Google sold domains to Squarespace which caused me to lose out on features. 

Create a free CF account. Point your Squarespace domain to the custom CF DNS name servers found on your account. And wait for the CF domain validation process to complete. It can take up to 24hrs.
Once this is done, CF is where all your A, AAA, CNAME, etc records will live and will be used. Anything in SS will NOT be used. 
And with CF you get access to features like DDNS & LetsEncrypt DNS challenges. As well as all the other features CF supports that SS doesn't.
I've been personally using this setup since the G Domain shutdown w/o any issues. This type of setup SS is my registrar, while CF is my DNS provider.
-edited for grammer/spelling
